### PR TITLE
ya, yazi: add pages

### PR DESCRIPTION
**Pull request by AnirudhG07**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by AnirudhG07** [comment]
Tldr pages for Yazi and ya. Note: This is known and approved by maintainer of yazi.

**Review by spageektti — COMMENTED** [review]
Hi. Thanks for your contribution. Please check my suggestions.

File: pages/common/yazi.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# yazi
+
+> Blazing fast terminal file manager written in Rust.
```
[the remark this answers is not in the document]
Comment (AnirudhG07) [reply pages/common/yazi.md:3]: It would better to keep the `Blazing Fast` since yazi is known for it.

[the remark this answers is not in the document]
Comment (spageektti) [reply pages/common/yazi.md:3]: But for me it sounds like opinion/ad

File: pages/common/yazi.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+- Launch yazi from current directory:
+
+`yazi`
+
+- Print debug information:
+
+`yazi --debug`
+
+- Write the cwd on exit to this file:
```
[the remark this answers is not in the document]
Comment (AnirudhG07) [reply pages/common/yazi.md:15]: should I change it to CWD?

**Review by spageektti — APPROVED** [review]

File: pages/common/yazi.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (AnirudhG07) [reply pages/common/yazi.md:3]: Check the repository, it's written there in the first line.

[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/common/yazi.md:3]: For descriptions, we use the same one as upstream. So I think mentioning it is fine, we do this for a few other pages too.

File: pages/common/yazi.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/common/yazi.md:15]: I think PWD (present working directly) would be more apt and known by UNIX, macOS and Linux users for this context.

File: pages/common/ya.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Subscribe to messages from all remote instances:
+
+`ya sub {{kinds}}`
+
+- Publish a message to the current instance with string body:
+
+`ya pub --str {{string_message}}`
+
+- Publish a message to the current instance with json body:
```
Comment (kbdharun) [inline pages/common/ya.md:22]:
```suggestion
- Publish a message to the current instance with JSON body:
```

File: pages/common/yazi.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (AnirudhG07) [reply pages/common/yazi.md:15]: Yazi works in all OS's, so I dont want to change the terminology used there.

File: pages/common/yazi.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# yazi
+
+> Blazing fast terminal file manager written in Rust.
+> Efficient, user-friendly, and customizable file management experience.
+> More information: <https://github.com/sxyazi/yazi>.
+
+- Launch Yazi from current directory:
```
Comment (kbdharun) [inline pages/common/yazi.md:7]:
```suggestion
- Launch Yazi from the current directory:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by AnirudhG07** [comment]
Done

**Pull request merged**